dispatcher/MongoConnect.py

```diff
@@ -273,7 +278,8 @@ def GetWantedState(self):
                         latest_settings[detector]['user'] = doc['user']
             self.latest_settings = latest_settings
             return self.latest_settings
-        except:
+        except Exception as e:
+            self.log.debug(f'GetWantedState failed of {type(e)} {e}')
```

Comment:
This function gets snake_cased in #23 (everything does, for that matter)

Reply:
nice